src/overlays/actors/ovl_En_Ma_Yto/z_en_ma_yto.h

```diff
@@ -40,7 +40,7 @@ typedef enum {
     /* 4 */ MA_YTO_TYPE_4  // HugCutscene? Doesn't seem to work properly in-game
 } EnMaYtoType;
 
-#define EN_MA_YTO_PARSE_TYPE(params) (((params)&0xF000) >> 12)
+#define EN_MA_YTO_PARSE_TYPE(thisx) (((thisx)->params & 0xF000) >> 12)
```

Comment:
How do you feel about removing `PARSE` from params macros?

Reply:
Sure, `GET` seems more common.